### PR TITLE
util/td-migration: Update scripts to support migration advanced features

### PR DESCRIPTION
**Pull request by ruomengh**
1. Pre-binding
2. Post-copy
3. Multi-stream

### CODE REVIEW

File: utils/td-migration/user-td.sh [hunk 1]
```diff
@@ -22,6 +22,11 @@ TELNET_PORT=9088
 INCOMING_PORT=6666
 CPU_NUM=2
 MEM_SIZE=8
+MIG_HASH="d83d9a38c238ef3b7bc207bbea3287a8b37b37e731480a8d240d2a6953086c5ecbdf7ee4c72fec3a3e9d4a87f4f9b4fe"
```
Comment (LeiZhou-97) [inline utils/td-migration/user-td.sh:25]: how about add one more parameter for mig_hash?

mig_hash may different, if use different config.

Comment (ruomengh) [reply utils/td-migration/user-td.sh:25]: This needs to be enhanced later after more hash is supported. Currently this one is used by default.

**Review by LeiZhou-97 — APPROVED** [review]
LGTM.

File: utils/td-migration/mig-td.sh [hunk 2]
```diff
@@ -9,6 +9,8 @@ else
     QEMU_EXEC="/usr/libexec/qemu-kvm"
 fi
 
+# QEMU_EXEC="/home/xiaocheng/ws/qemu-bkc/build/qemu-system-x86_64"
```
Comment (kenplusplus) [inline utils/td-migration/mig-td.sh:12]: Please remove comment-out code.

**Review by kenplusplus — CHANGES_REQUESTED** [review]

**Comment by ruomengh** [comment]
Close the PR and rebase in another PR.

**Pull request closed without merging**